### PR TITLE
Fix issue of memory leak in real-time monitoring page of Sentinel dashboard

### PR DESCRIPTION
**Pull request by zhangkai253**
fix issue of memory leak about realtime monitor page

### CODE REVIEW

**Comment by sczyh30** [comment]
Could you please describe your design in the PR description?

**Comment by zhangkai253** [comment]
> Could you please describe your design in the PR description?

just revoke useless charts positively, otherwise charts will accumulate more and more

**Comment by sczyh30** [comment]
> > Could you please describe your design in the PR description?
> 
> just revoke useless charts positively, otherwise charts will accumulate more and more

Nice. Could you please add a brief comment in your code?

File: sentinel-dashboard/src/main/webapp/resources/app/scripts/controllers/metric.js [hunk 1]
```diff
@@ -59,6 +59,10 @@ app.controller('MetricCtl', ['$scope', '$stateParams', 'MetricService', '$interv
       $interval.cancel(intervalId);
     });
     $scope.initAllChart = function () {
+      while($scope.charts.length > 0) {
+      	let chart = $scope.charts.pop();
+      	chart.destroy();
+      }
```
Comment (zhangkai253) [inline sentinel-dashboard/src/main/webapp/resources/app/scripts/controllers/metric.js:65]: revoke charts positively here

**Comment by zhangkai253** [comment]
> > > Could you please describe your design in the PR description?
> > 
> > 
> > just revoke useless charts positively, otherwise charts will accumulate more and more
> 
> Nice. Could you please add a brief comment in your code?

have done

**Review by sczyh30 — APPROVED** [review]
LGTM

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**